controllers/articles.js

```diff
@@ -0,0 +1,7 @@
+const {selectArticle} = require("../models/articles")
+
+exports.getArticles = (req,res,next)=>{
+    selectArticle().then((article)=>{
+        res.status(200).send({article: article})
```

Comment:
I'd change the key to be plural here simply because we are sending back an array of multiple article**s**, and later on we are going to have an endpoint that gets a single article which might get confusing